lib/resources/example_json/r4_examples_organization.rb

```diff
@@ -213,5 +213,153 @@ module Resources
         }
       ]
     }.freeze
+
+    R4_ORGANIZATION_CAREGIVER_SEARCH_BUNDLE ||= {
+      'resourceType': 'Bundle',
+      'id': '9cdc9b0a-1f7f-4a33-8c57-76514a7c7cdb',
+      'type': 'searchset',
+      'total': 2,
+      'link': [
+        {
+          'relation': 'self',
+          'url': 'https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Organization/$get-cg-for-mrcu?_id=3172822'
+        }
+      ],
+      'entry': [
+        {
+          'fullUrl': 'https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Organization/3172821',
+          'resource': {
+            'resourceType': 'Organization',
+            'id': '3172821',
+            'meta': {
+              'versionId': '4',
+              'lastUpdated': '2020-02-20T06:44:22Z'
+            },
+            'text': {
+              'status': 'generated',
+              'div': '<div xmlns=\"http://www.w3.org/1999/xhtml\"><p><b>Organization</b></p><p><b>Name</b>: '\
+                     'CH Caregiver</p><p><b>Status</b>: Active</p></div>'
+            },
+            'identifier': [
+              {
+                'extension': [
+                  {
+                    'valueCode': 'unknown',
+                    'url': 'http://hl7.org/fhir/StructureDefinition/data-absent-reason'
+                  }
+                ]
+              }
```

Comment:
Can we confirm that the data we see for these extension fields is always going to be data-absent-reason ? i think it would nice to sample out some real data rather than DAR.

Reply:
Updated here: 095808cbaf07c27e9de8ce1208dc3b1d3e835e04